pubpub/pubpub
# fix: add language to structuredCite callsite

Fixes a bug I accidentally introduced where Harvard-style pubs would fail to load because citation.js would error on some CSLs without a `lang` set.

_Test_

- Set a Pub with an inline cite to Harvard style
- Refresh and make sure it loads.

@idreyn feel free to merge and promote with your other citation pr.
